Wikipedia/Code/WMFAuthenticationManager.swift

```diff
@@ -177,21 +177,9 @@ class WMFAuthenticationManager: NSObject {
      *  Logs out any authenticated user and clears out any associated cookies
      */
     @objc public func logout(completion: @escaping () -> Void = {}){
-        self.resetLocalUserLoginSettings()
-        completion()
-    }
-    
-    @objc public func deleteLoginTokensAndBrowserCookies() {
```

Comment:
Why is it ok to kill this method? Does it not need to happen or does it happen in another way now?

Reply:
I moved it back where it was, to `logout`, so there's no need for a separate method anymore